packages/prisma-client/prisma/schema.prisma

```diff
@@ -3,16 +3,17 @@
 
 generator client {
   provider        = "prisma-client-js"
-  previewFeatures = ["views", "clientExtensions", "jsonProtocol"]
+  previewFeatures = ["views"]
   // see commands.ts
   // <output-placeholder-for-migrations>
   output          = "../src/__generated__"
   // </output-placeholder-for-migrations>
 }
 
 datasource db {
-  provider = "postgres"
-  url      = env("DATABASE_URL")
+  provider  = "postgres"
+  url       = env("DATABASE_URL")
+  directUrl = env("DIRECT_URL")
```

Comment:
this env variable should have been called DATABSE_DIRECT_URL  or PRISMA_DIRECT_URL .... you are in a global scope there, DIRECT_URL semantically doesn't make sense